diff --git a/app/src/views/webs/commons/users/login.tmpl b/app/src/views/webs/commons/users/login.tmpl
--- a/app/src/views/webs/commons/users/login.tmpl
+++ b/app/src/views/webs/commons/users/login.tmpl
@@ -29,9 +29,11 @@
 
     </form>
 
+<!--
     <h2><% gt "No account yet?" %></h2> 
     <div id="ClickHereToRegister">
         <a href="/users/register-new" class="registerButton"><% gt "Register" %></a>
     </div>
+-->
     <% include end_module() %>
 <% end template %>
diff --git a/app/src/views/webs/tatoeba/layouts/master.tmpl b/app/src/views/webs/tatoeba/layouts/master.tmpl
--- a/app/src/views/webs/tatoeba/layouts/master.tmpl
+++ b/app/src/views/webs/tatoeba/layouts/master.tmpl
@@ -54,11 +54,13 @@
             <ul>
             <% if not currentUserHelper.is_logged() %>
                 <!-- TODO put here the fancy login box -->
+<!--
                     <li class="menuSection" >
                         <a href="/users/register-new" >
                             <% gt "Register" %>
                         </a>
                     </li>
+-->
 
                     <li class="menuSection" >
                         <a href="/users/login" >
